types/aws-lambda/trigger/iot.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Handler } from "../handler";
+
+export type IoTHandler = Handler<IoTEvent, void>;
+
+// IoT
+// https://docs.aws.amazon.com/lambda/latest/dg/services-iot.html
+// IoT payload is not restriced to JSON, but JSON is highly recommended. Types as string, number or array are possible to use.
+
+export type IoTEvent = string | number | { [key: string]: any } | any[];
```

Comment:
Great feedback!
Maybe it can then be simplfied even more to:
```
export type IoTEvent<T = never> = string | number | T;
```
If using `T` then it can represent both object or array based on the type that is explicit put in. 

Example of use:
```
const event: IoTEvent<string[]> = ["1","2","3"];
const event: IoTEvent<{[key: string]: any}> = {
 a: 100,
 b: "string"
};

```

What do you think of this solution?

Reply:
Do it.

Reply:
Done, thanks :-)